res/res_pjsip/pjsip_config.xml

```diff
@@ -3027,7 +3068,7 @@
 					</since>
 					<synopsis>Number of seconds before an idle thread should be disposed of.</synopsis>
 				</configOption>
-				<configOption name="threadpool_max_size" default="0">
+				<configOption name="threadpool_max_size" default="50">
```

Comment:
I think it should be noted that this is deprecated or is a synonym for taskpool_max_size.